### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by capjamesg**
# Description

This PR includes a restructure of the Roboflow Inference documentation.

## Type of change

This change only affects documentation and does not modify Inference code.

## How has this change been tested, please provide a testcase or example of how you tested the change?

This change was tested by opening `http://localhost:8000` and checking that each page was formatted correctly.

## Any specific deployment considerations

The documentation will be deployed when this PR is merge to main.

## Docs

See above.

### CODE REVIEW

File: docs/foundation/doctr.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+[DocTR](https://github.com/mindee/doctr) is an Optical Character Recognition model.
+
+You can use DocTR with Inference to identify and recognize characters in images.
+
+### How to Use DocTR
+
+To use DocTR with Inference, you will need a Roboflow API key. If you don't already have a Roboflow account, [sign up for a free Roboflow account](https://app.roboflow.com). Then, retrieve your API key from the Roboflow dashboard. Run the following command to set your API key in your coding environment:
+
+```
```
Comment (paulguerrie) [inline docs/foundation/doctr.md:9]: At this point, have we talked about how to start the roboflow inference server docker container? If now, we should mention it here and link to that part of the docs.

File: docs/quickstart/explore_models.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+
+## Run a Private, Fine-Tuned Model
+
+You can run models you have trained privately on Roboflow with Inference. To do so, first go to your [Roboflow dashboard](https://app.roboflow.com).  Then, choose the model you want to run.
+
+![Roboflow dashboard](https://media.roboflow.com/docs-models.png)
+
+Click the "Deploy" link in the sidebar to find the information you will need to use your model with Inference:
+
+![Model list](https://media.roboflow.com/docs-sidebar-list.png)
```
Comment (paulguerrie) [inline docs/quickstart/explore_models.md:80]: This image appears very large. Any way to make it smaller?

File: docs/quickstart/run_model_on_image.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+pip install roboflow
+```
+
+Next, create a new Python file and add the following code:
+
+```python
+from roboflow import Roboflow
+rf = Roboflow(api_key="API_KEY")
+project = rf.workspace().project("MODEL_ENDPOINT")
+model = project.version(VERSION, local="http://localhost:9001").model
```
Comment (paulguerrie) [inline docs/quickstart/run_model_on_image.md:22]: We use localhost here but we haven't taught the user how to start a server yet. Can we add that snippet? Or better yet, can we just show how to run inference on an image the most simple way?

```suggestion
from inference.models.utils import get_roboflow_model
model = get_roboflow_model(...)
result = model.infer(image) #image can be a url string, numpy array, pil image, etc.
```

File: docs/quickstart/run_model_on_rtsp.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
+You can run computer vision models on RTSP stream frames with Inference.
```
Comment (paulguerrie) [inline docs/quickstart/run_model_on_rtsp.md:1]: This guide is nearly identical to the webcam guide. Should we combine them into a single guide?

File: docs/foundation/about.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
+Foundation models are machine learning models that have been trained on vast amounts of data to accomplish a specific task.
+
+For example, OpenAI trained CLIP, a foundation model. CLIP enables you to classify images. You can also compare the similarity of images and text with CLIP.
+
+The CLIP training process, which was run using over 400 million pairs of images and text, allowed the model to build an extensive range of knowledge, which can be applied to a range of domains.
+
+Foundation models are being built for a range of vision tasks, from image segmentation to classification to zero-shot object detection.
+
+Autodistill supports the following foundation models:
```
Comment (paulguerrie) [inline docs/foundation/about.md:9]: Typo? Autostill --> Inference

File: docs/foundation/about.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+- LC2S-Net: Detect the direction in which someone is looking.
```
Comment (paulguerrie) [inline docs/foundation/about.md:11]: I think we call this Gaze.

File: docs/quickstart/run_model_on_video.md [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+
+To use fine-tuned with Inference, you will need a Roboflow API key. If you don't already have a Roboflow account, [sign up for a free Roboflow account](https://app.roboflow.com). Then, retrieve your API key from the Roboflow dashboard. Run the following command to set your API key in your coding environment:
+
+```
+export API_KEY=<your api key>
+```
+
+Create a new Python file and add the following code:
+
+```python
```
[the remark this answers is not in the document]
Comment (capjamesg) [reply docs/quickstart/run_model_on_video.md:16]: Do you have a code snippet? I don't have any examples to hand.

[the remark this answers is not in the document]
Comment (paulguerrie) [reply docs/quickstart/run_model_on_video.md:16]: You can just pass the path of a video file or the url of a hosted video file to the `source` parameter.

**Pull request merged**